R/iregnet.R

```diff
@@ -116,10 +117,10 @@
 #' Paths for Cox's Proportional Hazards Model via Coordinate Descent, Journal
 #' of Statistical Software, Vol. 39(5) 1-13
 #' \url{http://www.jstatsoft.org/v39/i05/}
-#'
+#' @useDynLib iregnet
```

Comment:
using this here means that roxygen2 will generate the useDynLib line in NAMESPACE